dead-letter.tf

```diff
@@ -0,0 +1,24 @@
+// single alert to minify unnecessary cost because threshold used in here is minimal
+module "bulk-scan-dead-letter-alert" {
+  source            = "git@github.com:hmcts/cnp-module-metric-alert"
+  location          = "${azurerm_application_insights.appinsights.location}"
+  app_insights_name = "${azurerm_application_insights.appinsights.name}"
+
+  enabled    = "${var.env == "prod"}"
+  alert_name = "Bulk_Scan_DLQ_-_BSP"
+  alert_desc = "Triggers when bulk scan services record at least one dead lettered message within a 15 minutes window timeframe."
+
+  app_insights_query = <<EOF
+exceptions
```

Comment:
```
union exceptions, trace 
| where * contains "dead-letter"
```
covers more cases?

Reply:
I wanted to avoid logging EnvelopeMessageProcessor deadletter twice as Appinsight custom event logs it as well . If that's acceptable than I will change it.

Reply:
Added traces as well.